cartography/data/indexes.cypher

```diff
@@ -74,6 +74,15 @@ CREATE INDEX ON :LoadBalancerV2(dnsname);
 CREATE INDEX ON :LoadBalancerV2(id);
 CREATE INDEX ON :NetworkInterface(id);
 CREATE INDEX ON :NameServer(id);
+CREATE INDEX ON :OCICompartment(ocid);
+CREATE INDEX ON :OCICompartment(name);
+CREATE INDEX ON :OCIGroup(ocid);
```

Comment:
Echoing Sacha's comment here on using `id` as an index. Let me know if this doesn't make sense.